spec/javascript/pages/Offer.spec.js

```diff
@@ -35,6 +35,10 @@ describe('Offer', () => {
     person: $person,
   }})
 
+  def('submission', () => { return {
+    listing: $offer,
```

Comment:
is this $offer implicit bc of import Offer ?

Reply:
There's a `def('offer' ...` a couple lines above, probably was collapsed in the diff. This is how `bdd-lazy-var` does the equivalent of rspec's `let` in JS specs.